src/lib.rs

```diff
@@ -24,15 +26,35 @@ pub fn package(name: &str) -> Option<&cargo_metadata::Package> {
     metadata().packages.iter().find(|x| x.name == name)
 }
 
+pub fn default_build_dir(release: bool) -> &'static camino::Utf8PathBuf {
```

Comment:
Use slices when possible. No ref to owned types. It's like `&String` vs `&str` or `&Vec<_>` vs `&[_]`.

```suggestion
pub fn default_build_dir(release: bool) -> &'static camino::Utf8Path {
```